src/backend/marsha/core/management/commands/sync_medialive_video.py

```diff
@@ -85,17 +89,24 @@ def handle(self, *args, **options):
             # the channel name contains the environment, the primary key and the created_at
             # stamp. Here we want to use the primary key
             _environment, live_pk, _stamp = medialive_channel["Name"].split("_")
-            live = Video.objects.get(pk=live_pk)
-            self.stdout.write(f"Checking video {live.id}")
-            channel_state = medialive_channel.get("State").casefold()
-
-            # If the live state in not sync with media channel state,
-            # we manually update to avoid soft lock
-            live_state = live.live_state.casefold()
-            if not self.is_channel_sync_with_video(live_state, channel_state):
+            live = Video.objects.filter(pk=live_pk).first()
```

Comment:
keeping `live = Video.objects.get(pk=live_pk)` with a try/except is more elegant IMO.